pip version of pysam 0.15.1 seemingly not supported in Mac OS

In Mac OS, error occurs when running pypi installation of pysam
```
htslib/cram/cram_io.c:63:10: fatal error: 'os/lzma_stub.h' file not found
      #include "os/lzma_stub.h"
               ^~~~~~~~~~~~~~~~
      1 error generated.
      error: command 'clang' failed with exit status 1
```

Issue in pysam https://github.com/pysam-developers/pysam/issues/645 does not give much options apart from using pysam 0.13. 

Is it possible that the same error causes installation of pysam 0.13 instead of 0.15.1 via conda? (this is how I understand the reason we use pypi instead of conda for pysam installation)

Related to #10

--- a/environment.yml
+++ b/environment.yml
@@ -91,7 +91,7 @@ dependencies:
   - zict=0.1.3
   - pip:
     # version discrepancy for pysam on bioconda, install from pypi to be sure
-    - pysam==0.15.1
+    - pysam==0.15.2
     - pysamstats==1.1.2
     # not on conda-forge
     - anhima==0.11.2

